src/components/fields/location-field/location-helper.ts

```diff
@@ -45,7 +45,8 @@ export namespace LocationHelper {
 		mustHavePostalCode: boolean,
 		reverseGeocodeAborter: MutableRefObject<AbortController>,
 		onError: (error: any) => void,
-		excludeNonSG?: boolean
+		excludeNonSG?: boolean,
+		bufferRadius = 500
```

Comment:
according to [OneMap api docs](https://www.onemap.gov.sg/apidocs/reversegeocode/#svy21), the acceptable range is 0-500, we will need to restrict the value.

if a bufferRadius beyond 0-500 is provided, we constrain to that range and do a `console.warn` to inform the consumer an invalid value is specified.